python/cudf/cudf/core/column/numerical.py

```diff
@@ -105,6 +105,14 @@ def binary_operator(self, binop, rhs, reflect=False):
     def _apply_scan_op(self, op):
         return libcudf.reduce.scan(op, self, True)
 
+    @property
+    def nan_count(self):
+        if self.dtype.kind == "f":
+            s = cudf.Series(libcudf.unary.is_nan(self))
+            return len(s[s])
```

Comment:
May be better off doing a sum instead of a boolean masking here.

Reply:
We now wouldn't need this function, so removed it.